app/views/postal_addresses/edit.html.erb

```diff
@@ -0,0 +1,69 @@
+<% content_for(:page_title, t('service.title', page_title: "Edit postal address for #{@booking_request.name}")) %>
+
+<div class="page-header">
+  <ol class="breadcrumb">
+    <li><a href="<%= root_path %>">Appointment planner</a></li>
+    <% if @booking_request.appointment %>
+      <li><a href="<%= appointments_path %>">Appointments</a></li>
+      <li><a href="<%= edit_appointment_path(@booking_request.appointment) %>"><%= @booking_request.name %></a></li>
+    <% else %>
+      <li><a href="<%= booking_requests_path %>">Booking requests</a></li>
+      <li><a href="<%= new_booking_request_appointment_path(@booking_request) %>"><%= @booking_request.name %></a></li>
+    <% end %>
+    <li class="active">Edit postal address</li>
+  </ol>
+
+  <h1>
+    Edit postal address for <%= @booking_request.name %><br>
+    <small>Booking reference: <%= @booking_request.reference %></small>
+  </h1>
+</div>
+
+<%= form_for @postal_address, url: booking_request_postal_address_path(@booking_request), as: :postal_address, method: :patch do |f| %>
+  <div class="row">
+    <div class="col-md-6 l-booking-request-details">
+      <% if @postal_address.errors.any? %>
+        <div class="alert alert-danger t-errors" role="alert">
+          <h3 class="alert__heading h4">There's a problem</h3>
+          <ul>
+            <% @postal_address.errors.full_messages.each do |msg| %>
+              <li><%= msg %></li>
+            <% end %>
+          </ul>
+        </div>
+      <% end %>
+      <div class="row">
+        <div class="col-md-12">
+          <div class="form-group">
+            <%= f.label :address, for: :postal_address_address_line_one %>
+            <%= f.text_field :address_line_one, class: 'form-control t-address-line-one' %>
```

Comment:
Can these be placed in separate form groups so they are spaced out a little? I see the intention here but elsewhere in the platform, wherever we take address lines we space the fields consistently.